### DISCUSSION TITLE
Liquify tool

### DISCUSSION THREAD

**Discussion by Omburliu** [keiwando/toonsquid#537] [Ideas]
Will you guys consider adding the liquify tool to the app in the future?
Really need these kinda small adjustments for animation or animatics but selecting and moving parts using the wrap tool every time is a little inconvenient.

**Comment by Lioncub901** [comment]
Nice to add

**Comment by keiwando** [comment]
ToonSquid now has a Liquify feature. You can read more about it here:

https://toonsquid.com/handbook/brushes/liquify/

It was added with the 2.0.22 update:

https://toonsquid.com/updates/ToonSquid-2_0_22/
https://www.youtube.com/watch?v=UV3eiO2tdhI
